### PR TITLE
Ra 1197

### PR DESCRIPTION
**Pull request by vmartos**
#### Things to review:
- [ ] Unit tests
- [ ] No hardcoded resources (strings, sizes, colors...)
- [ ] Sorted usings
- [ ] Add summaries where needed 

If needed, you can add new checkboxes to https://github.com/fluendo/VAS/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs [hunk 1]
```diff
@@ -26,28 +28,64 @@ namespace VAS.Core.ViewModel.Statistics
...
+				UpdateEmptySerie ();
+			}
 		}
 
 		/// <summary>
 		/// Gets the percent text value of the serie.
 		/// </summary>
 		/// <value>The percent text value.</value>
-		public string PercentValueText { get; private set; }
+		// public string PercentValueText { get => String.Format ("{0:0.0}%", PercentValue); }
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs]: Commented code

File: VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs [hunk 2]
```diff
@@ -26,28 +28,64 @@ namespace VAS.Core.ViewModel.Statistics
...
+		/// <value>The empty serie.</value>
+		public SeriesVM EmptySerie { get; private set; }
+
+		void UpdateEmptySerie () {
+			EmptySerie.Elements = TotalElements - PercentSerie.Elements;
+		}
+
+		protected override void ForwardPropertyChanged (object sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName == "Elements" && sender == PercentSerie) {
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/PercentCircularChartVM.cs]: use `NeedsSync`

File: VAS.Core/ViewModel/Statistics/SeriesVM.cs [hunk 3]
```diff
@@ -28,7 +28,7 @@ public class SeriesVM : ViewModelBase
 	{
 		public SeriesVM () { }
 
-		public SeriesVM (string title, int elements, Color color)
+		public SeriesVM (string title, Color color, int elements = 0)
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/SeriesVM.cs]: Do we want a series with 0 elements?

File: VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs [hunk 4]
```diff
@@ -44,14 +45,17 @@ public sealed class TwoBarChartVM : BarChartVM
...
-			Series = new SeriesCollectionVM ();
-			Series.ViewModels.Add (leftSerie);
-			Series.ViewModels.Add (rightSerie ?? new SeriesVM ("empty", total - leftSerie.Elements, App.Current.Style.ThemeBase));
+			LeftSerie = leftSerie;
+			syncRightSerie = rightSerie == null;
+			RightSerie = rightSerie ?? new SeriesVM ("empty", App.Current.Style.ThemeBase);
+			Series.ViewModels.AddRange (new List<SeriesVM> { LeftSerie, RightSerie });
 			leftTextStyle = rightTextStyle = TwoBarTextStyle.Hidden;
-			TotalNumber = ReferenceNumber = total;
+			ReferenceNumber = TotalElements = total;
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs]: What does this change?

File: VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs [hunk 5]
```diff
@@ -173,6 +175,21 @@ void InternalUpdate (ref string text, SeriesVM serie, ref TwoBarTextStyle style)
 				break;
 			}
 		}
+
+		protected override void ForwardPropertyChanged (object sender, System.ComponentModel.PropertyChangedEventArgs e)
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs]: using

File: VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs [hunk 6]
```diff
@@ -173,6 +175,21 @@ void InternalUpdate (ref string text, SeriesVM serie, ref TwoBarTextStyle style)
...
+		{
+			if (sender == LeftSerie && e.PropertyName == "Elemets") {
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs]: Use `NeedsSync`

File: VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs [hunk 7]
```diff
@@ -173,6 +175,21 @@ void InternalUpdate (ref string text, SeriesVM serie, ref TwoBarTextStyle style)
...
+				SyncRightSerie ();
+			}
+
+			base.ForwardPropertyChanged (sender, e);
+		}
+
+		void SyncRightSerie () {
+			if (syncRightSerie) {
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/Statistics/TwoBarChartVM.cs]: This method is weird, why would we call this if `syncRightSerie`  is false?

File: VAS.UI.Helpers.Gtk2/ButtonHelper.cs [hunk 8]
```diff
@@ -212,6 +213,19 @@ public static void BindManually (this Button button, Command command, object par
...
+				toggleButtons.Add (button);
+			}
+			LinkToggleButtons (toggleButtons);
+		}
+
+		/// <summary>
+		/// Links a group of toggle buttons to act like a radio button group.
+		/// </summary>
+		/// <param name="buttons">Buttons.</param>
+		public static void LinkToggleButtons (IEnumerable<ToggleButton> buttons)
```
Comment (vguzmanp) [inline VAS.UI.Helpers.Gtk2/ButtonHelper.cs]: Why do we need this now?

File: VAS.Core/ViewModel/Statistics/SeriesVM.cs (hunk 3 above)
Comment (SandraPG85) [reply VAS.Core/ViewModel/Statistics/SeriesVM.cs]: this change means that you can set an initial value or not

File: VAS.UI.Helpers.Gtk2/ButtonHelper.cs (hunk 8 above)
Comment (SandraPG85) [reply VAS.UI.Helpers.Gtk2/ButtonHelper.cs]: This function existed with a params attribute, if you already have a list it is passed to this function.

**Pull request merged**